src/html/include/base.html

```diff
@@ -67,6 +67,7 @@
                     <li><a href="https://docs.spongepowered.org"><i class="fa-fw fa fa-book"></i>Docs</a></li>
                     <li><a href="https://jd.spongepowered.org"><i class="fa-fw fa fa-graduation-cap"></i>Javadocs</a></li>
                     <li><a href="https://forums.spongepowered.org/c/plugins/plugin-releases"><i class="fa-fw fa fa-plug"></i>Plugins</a></li>
+                    <li><a href="https://ore.spongepowered.org/"><img src="assets/img/icons/ore.svg" alt="" class="fa-fw fa ore-nav">Ore <sup>Beta</sup></a></li>
```

Comment:
Did you mean `ore-logo`?

Reply:
My bad, some repos already use `ore-logo` for a separate Ore logo.